src/index.js

```diff
@@ -135,7 +135,7 @@ export default async function microbundle(options) {
 	}
 
 	function formatSize(size, filename, type) {
-		const pretty = prettyBytes(size);
+		const pretty = options.raw ? `${size} B` : prettyBytes(size);
```

Comment:
@andrewiggins I'm wondering if we could get around even having to add the `--raw` option here - what if we just always show bytes for <5000b libs?

```js
const pretty = size < 5000 ? `${size} B` : prettyBytes(size)
```

Reply:
In order to get this to work, you'll likely need to move the option read to `getSizeInfo` now.

Also, the size passed to this function is pre-compressed, so I'd recommend getting the size directly from `code` in the function below.